packages/status-js/src/helpers/get-objects-difference.ts

```diff
@@ -0,0 +1,23 @@
+type Input<Value> = Record<string, Value>
+
+export function getObjectsDifference<Value>(
+  oldObject: Input<Value>,
+  newObject: Input<Value>
+) {
+  const added: Record<string, Value> = {}
```

Comment:
The `added` field currently returns the same structure it is accepting, because `add*` methods rely on keys being the ids.